lib/chef/knife/winrm_base.rb

```diff
@@ -114,7 +114,9 @@ def self.included(includer)
 
       def locate_config_value(key)
         key = key.to_sym
-        config[key] || Chef::Config[:knife][key]
+        value = config[key] || Chef::Config[:knife][key] || default_config[key]
```

Comment:
@btm Actually it doesn't.  If you drop into pry on a number of the unit tests off master (as it stands now), you'll see a bunch of config settings didn't resolve properly.